documentation/proposals/Proposal - 3.0 & 3.X Software Development Plan.md

```diff
@@ -0,0 +1,149 @@
+# Summary
+
+3.0 software development plan & ongoing monthly update, breaking change, and support policy for 3.X.
+
+# Contributors
+- Dylan P (@Perksey)
+
+# Current Status
+- [x] Proposed
+- [ ] Discussed with Working Group
+- [ ] Approved
+- [ ] Implemented
+
+# Silk.NET 3.0
+
+## Goals of 3.0
+
+The key tenets of 3.0 are **portability**, **maintainability**, **usability**, and **performance**. To this end, the following objectives have been identified:
+- Use .NET 6 - the first version of modern .NET to run on the majority of our desired target platforms
+    - (tenet: portability) 
+- Allow Silk.NET's rich abstractions to be integrated into other frameworks rather than being completely standalone.
+    - In order of priority: WPF, WinForms, MAUI, Avalonia (based on demand). (tenet: usability) 
```

Comment:
Weird to list them like this. I'd remove this.

Reply:
True but like order of priority. Idk about that, I'd argue the priorities are different.

Reply:
True, I suppose it's subjective. Will remove the in order of priority and just know in my heart that is the order of priority :P 

```suggestion
    - WPF, WinForms, MAUI, Avalonia (tenet: usability) 
```